### ISSUE TITLE
Post-action is always skipped

### ISSUE THREAD

**Issue by ofalvai** [earthly/actions-setup#25]
I can't make the post-action cache save work no matter what I try.

This is my workflow:

```yaml
jobs:
  build:
    runs-on: ubuntu-latest
    env:
      FORCE_COLOR: 1
    steps:
      - name: Install Earthly
        uses: earthly/actions-setup@v1
        with:
          version: latest
          use-cache: 'true'

      - uses: actions/checkout@v3

      - name: Run build
        run: earthly --ci +build-all
```

When this job runs, the post-action is always skipped. I also enabled debug logging and this is the output of the post-action:

```
##[debug]Evaluating condition for step: 'Post Install Earthly'
##[debug]Evaluating: (inputs.use-cache == 'true')
##[debug]Evaluating Equal:
##[debug]..Evaluating Index:
##[debug]....Evaluating inputs:
##[debug]....=> Object
##[debug]....Evaluating String:
##[debug]....=> 'use-cache'
##[debug]..=> null
##[debug]..Evaluating String:
##[debug]..=> 'true'
##[debug]=> false
##[debug]Expanded: (null == 'true')
##[debug]Result: false
```

Is there anything I'm doing wrong?

**Comment by alexcb** [comment]
I wonder if `use-cache` is a valid name? usually `-` is reserved for subtraction.

looks like https://github.com/earthly/actions-setup/pull/13#issuecomment-1472998129 was missed.

Can you open a PR to rename this to something that's valid?

**Comment by ofalvai** [comment]
Sure thing @alexcb, I opened a PR: https://github.com/earthly/actions-setup/pull/27

First, I'd rather try solving this without a breaking change (renaming the input), so I surrounded the expression with `${{ }}`, let's see if it works. Can you please approve the workflow run to see if that works?

**Comment by alexcb** [comment]
fixed in #28
